### ISSUE TITLE
Add custom optimizer schedule

### ISSUE THREAD

**Issue by 9autilus** [govindnh4cl/squeezenet#68]
We decreased the learning rate manually over a period of couple of days while training. The check-in code would not be able to train the model due to a fixed learning rate.
